0.18.0: python import errors

When updating to 0.18.0, we are unable to import `opentimelineio` and receive the following error:

```
  src/de_core/common/edl_parser.py:3: in <module>
      import opentimelineio as otio
  ../../../../.venv/lib/python3.12/site-packages/opentimelineio/__init__.py:14: in <module>
      from . import (
  ../../../../.venv/lib/python3.12/site-packages/opentimelineio/opentime.py:4: in <module>
      from . _opentime import ( # noqa
  E   ModuleNotFoundError: No module named 'opentimelineio._opentime'
```

--- a/.github/workflows/python-package.yml
+++ b/.github/workflows/python-package.yml
@@ -187,7 +187,7 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Build wheels (Python 3)
-        uses: pypa/cibuildwheel@v2.22.0
+        uses: pypa/cibuildwheel@v3.2.1
         with:
           output-dir: wheelhouse
         env:

